### PR TITLE
[US.1][2] Object representation of the board + test

### PR DESCRIPTION
**Pull request by wektor1**
#6 SP:3

### CODE REVIEW

**Comment by just44play** [comment]
Test doesn't assert anything... ?

**Comment by just44play** [comment]
@LordLukin some tip, what should be tested?

File: src/Board.hpp [hunk 1]
```diff
@@ -0,0 +1,16 @@
+#pragma once
+#include <vector>
+
+class Board
+{
+    const int row_number_, col_number_;
```
Comment (LordLukin) [inline src/Board.hpp]: Please move col_number to the next line.
Do we want a snake_case naming convention? In C++ usually CamelCase is used, but I do not insist.

File: src/Board.cpp [hunk 2]
```diff
@@ -0,0 +1,16 @@
+#include "Board.hpp"
+
+Board::Board(int row_number, int col_number,
```
Comment (LordLukin) [inline src/Board.cpp]: 1 param per line is the best formatting, so please move col_number to the next line

File: src/Board.hpp [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+    std::vector<std::vector<int>> rows_;
```
Comment (LordLukin) [inline src/Board.hpp]: There should be some aliases like
```
using Line = std::vector<int>;
using Lines = std::vector<Line>;
```

**Comment by LordLukin** [comment]
> @LordLukin some tips, what should be tested?

If we have only constructor then creating a Board is enough. New tests will be added when this class grows.

**Comment by LordLukin** [comment]
Who else can review this code?

**Pull request merged**